README.md

```diff
@@ -93,6 +95,11 @@ Note: * Planning
   
   By default, the Kmesh base function is used, other function can be selected by adjusting the startup parameters in the yaml file.
   
+  - Install from Helm

```

Comment:
Pls place helm install description before yaml install description, and optimized yaml install description.